curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/6401cceab4b39e4639d177e4.md

```diff
@@ -0,0 +1,162 @@
+---
+id: 6401cceab4b39e4639d177e4
+title: Step 19
+challengeType: 0
+dashedName: step-19
+---
+
+# --description--
+
+We now have everything we want to include in the UI. What we have to do next is to make the "Load More Authors" button work so we can fetch more authors. You can do this by adding a `click` event to the "Load More Authors" button, then run a function that will fetch 8 more authors whenever the button is clicked.
```

Comment:
We will need to remove all references to first person narrative per the [docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions) here